mungegithub/mungers/submit-queue.go

```diff
@@ -72,7 +72,7 @@ const (
 var (
 	_ = fmt.Print
 	// This MUST cause a RETEST of everything in the sq.RequiredRetestContexts
-	retestBody = fmt.Sprintf("@%s test this [submit-queue is verifying that this PR is safe to merge]", jenkinsBotName)
+	retestBody = fmt.Sprintf("/test all [submit-queue is verifying that this PR is safe to merge]")
```

Comment:
This is the comment that tomorrow's submit queue will leave to trigger testing before merging.

We need to ensure that all AlwaysRun jobs trigger on this comment as well.